fixtures/build/dir012/.env

```diff
@@ -1,5 +1,5 @@
 # In all environments, the following files are loaded if they exist,
-# the later taking precedence over the former:
+# the latter taking precedence over the former:
```

Comment:
pretty sure that's a typo

Reply:
Well, I applied a recipe update -> https://github.com/symfony/recipes/blob/master/symfony/flex/1.0/.env